components/brave_wallet_ui/components/extension/encryption-key-panel/index.tsx

```diff
@@ -69,7 +71,7 @@ function EncryptionKeyPanel (props: Props) {
       <PanelTitle>
         {panelType === 'request'
           ? getLocale('braveWalletProvideEncryptionKeyTitle')
-          : getLocale('braveWalletReadEncryptedMessageTitle').replace('$1', encryptionKeyPayload.origin.url)}
+          : getLocale('braveWalletReadEncryptedMessageTitle').replace('$1', (encryptionKeyPayload || decryptPayload).origin.url)}
```

Comment:
Since this is already being checked for `panelType === 'request'` we can just use `decryptPayload.origin.url` for the string `replace`

Reply:
Fixed in 8999423ff185b6a3e87fa42ac28dfe06e34a6c36